services/uploads/serverless.yml

```diff
@@ -112,7 +112,7 @@ resources:
             - Action:
                 - "s3:GetObject"
               Effect: "Deny"
-              Resource: !Sub ${AttachmentsBucket.Arn}/*
+              Resource: !Sub ${AttachmentsBucket.Arn}/*.html
```

Comment:
Don't modify this statement.  This statement is saying "deny any request to download a file that is not clean".  We want to leave that intact. 

You want to add a NEW statement, saying "deny any request for a *.html file"